sdk/storage/storage-internal-avro/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@azure/storage-internal-avro",
+  "sideEffect": false,
+  "private": true,
+  "author": "Microsoft Corporation",
+  "version": "1.0.0-preview.1",
+  "description": "internal avro parser",
+  "license": "MIT",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "main": "./srt/index.ts",
+  "module": "dist-esm/index.js",
+  "types": "./types/index.d.ts",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "scripts": {
+    "build:es6": "tsc -p tsconfig.json",
+    "build:nodebrowser": "rollup -c 2>&1",
+    "build": "npm run build:es6 && npm run build:nodebrowser",
+    "build:test": "npm run build:es6 && rollup -c rollup.test.config.js 2>&1",
+    "clean": "rimraf dist dist-esm dist-test typings temp dist-browser/*.js* dist-browser/*.zip statistics.html coverage coverage-browser .nyc_output *.tgz *.log test*.xml TEST*.xml",
+    "unit-test:node": "mocha --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --full-trace -t 120000 dist-test/index.node.js",
+    "test:node": "npm run clean && npm run build:test && npm run unit-test:node",
+    "format": "prettier --write --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\""
+  },
+  "dependencies": {
+    "tslib": "^1.10.0"
+  },
+  "devDependencies": {
+    "@azure/identity": "^1.1.0-preview",
```

Comment:
Why Avro parser depends on @azure/identity? All dev dependencies are required here?

Reply:
Nope. Will clean up the dependency when things finiliaze.
And this package.json is only for test usage and won't have any effect when releasing.